server/paths/apply.ts

```diff
@@ -21,6 +21,7 @@ const paths = {
     },
     update: singleApplicationPath,
     appendToList: singleApplicationPath.path('list'),
+    removeFromList: singleApplicationPath.path('list').path('remove'),
```

Comment:
I'm not sure what the endpoint should be, this doesn't feel right but couldn't think of what else 🤔

Reply:
maybe would be clearer to make this ACCT specific? e.g. `singleApplicationPath.path('acct-notes').path('remove')`

Reply:
does it not still make sense to differentiate paths for different add/remove to list tasks? E.g. so we can pick up more useful and granular data from audit events...?

Reply:
hmm yeah that's interesting point and has made me realise we aren't sending over the query param data to the audit service for these application methods (`appendToList` `removeFromList` and `update`) as I thought we were...

Will have a look at some options